A/AcousticsToolbox/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "AcousticsToolbox"
+version = v"2020.11.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://oalib.hlsresearch.com/AcousticsToolbox/at_2020_11_4.zip", "6f9fe5d3d7fd99fc34fb2a2c3308f2e660a1cc0ea93f0651111af698acf442f0")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+rm -rf __MACOSX
+cd at_2020_11_4
+perl -p -i -e 's/\-march=native//; s/\-ffast\-math//; s/\-mtune=native//;' Makefile
+make
+mkdir -p $bindir
+find . -name *.exe -exec cp {} $bindir \;
```

Comment:
This is finding, and installing, a binary file that comes in the zipball:
```console
%  find . -name '*.exe'
./RAM/ram.exe
./RAM off oalib/ram.exe
./RAM_ljh/ram.exe
```

Reply:
Good find. I added a statement to delete pre-packaged exe files now:
```
find . -name *.exe -exec rm {} \;
```
before `make`.